### PR TITLE
api-gateway: some code cleanup

### PR DESCRIPTION
**Pull request by TeodorRussu**
api-gateway: some code cleanup

### CODE REVIEW

**Review by iluwatar — CHANGES_REQUESTED** [review]
The build is failing due to Checkstyle errors

**Review by ohbus — APPROVED** [review]

**Comment by ohbus** [comment]
I cannot confirm the build status.

**Comment by ohbus** [comment]
@TeodorRussu Can you check the [documentation from here](https://github.com/iluwatar/java-design-patterns/wiki/03.-Coding-conventions#formatting) and update the CheckStyle issue of this PR to get it accepted!

File: api-gateway/api-gateway-service/src/main/java/com/iluwatar/api/gateway/ApiGateway.java [hunk 1]
```diff
@@ -45,7 +47,7 @@
    *
    * @return Product information for clients on a desktop
    */
-  @RequestMapping(path = "/desktop", method = RequestMethod.GET)
+  @GetMapping(path = "/desktop")
```
Comment (ravening) [inline api-gateway/api-gateway-service/src/main/java/com/iluwatar/api/gateway/ApiGateway.java:50]: you can just do `@GetMapping("/desktop")`

**Comment by ohbus** [comment]
@TeodorRussu please look into the comments by @ravening as they will mostly be reported by _sonarbot_.

As this PR has been inactive for a long time, we will close it soon if we do not see any updates.

**Comment by TeodorRussu** [comment]
not today

**Pull request closed without merging**